e3db/types/search_result.py

```diff
@@ -0,0 +1,118 @@
+from .search import Search
+from .record import Record
+
+class SearchResult(object):
+
+    def __init__(self, search, records, after_index=0, search_id=""):
+        """
+        Initialize the SearchResult class.
+
+        This object type is used as a storage for the results of v2 Search query.
+        The main purpose is to allow iteration over a list of returned records
+        that are the result of a Search operation.
+
+        Parameters
+        ----------
+        search : e3db.Search
+            Search object stored for later operations, such as looking up the
+            after index returned from the server during search execution.
+
+        records : list<e3db.Record>
+            A list of records to store for later iteration and operations.
+
+        after_index : int
+            Index that indicates where to start the next query for batch jobs.
+
+        search_id : str
+            ID to access search_id for async searches, but currently is not implemented in
+            the search service.  
```

Comment:
Same as other comment, remove Search Service in favor of either "server" or "E3DB"